federation/source-config-handlers/src/main/java/org/codice/ddf/admin/sources/wfs/WfsSourceUtils.java

```diff
@@ -69,9 +80,10 @@
      * @param password
      * @return report
      */
-    public static ProbeReport sendWfsCapabilitiesRequest(String url, String username,
+    public ProbeReport sendWfsCapabilitiesRequest(String url, String username,
             String password) {
-        ProbeReport requestResults = RequestUtils.sendGetRequest(url, username, password);
+        String reqUrl = url + GET_CAPABILITIES_PARAMS + ACCEPT_VERSION_PARAMS;
```

Comment:
Personal preference: I would use `String.format` here instead of concatenation. It's basically a wash in a simple case like this, though.